compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -846,45 +856,27 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
         ret
     }
 
-    fn lower_attrs(&mut self, id: HirId, attrs: &[Attribute]) -> &'hir [hir::Attribute] {
+    fn lower_attrs(
```

Comment:
lower_attrs is changed, because it will now call into the `rustc_attr_parsing::AttributeParser`. It needs a tiny bit more information too, like the span of the target ast node, which is why a few other things are changed in this crate.